### PR TITLE
testing/pdal: new aport

### PR DESCRIPTION
**Pull request by chambbj**
https://www.pdal.io
PDAL is a C++ BSD library for translating and manipulating point cloud data.

### CODE REVIEW

File: testing/libspatialite/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=libspatialite
+pkgver=4.3.0
+_pkgver="$pkgver"a
```
Comment (ncopa) [inline testing/libspatialite/APKBUILD]: Can we use `pkgver="4.3.0a"` instead of using `_pkgver`?

File: testing/libspatialite/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+pkgrel=0
+pkgdesc="SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities."
```
Comment (ncopa) [inline testing/libspatialite/APKBUILD]: Can we make the pkgdesc shorter? (max ~70 chars?)

File: testing/libspatialite/APKBUILD (hunk 1 above)
Comment (jirutka) [reply testing/libspatialite/APKBUILD]: What does "a" mean in this case - alpha or like in openssl?

Comment (chambbj) [reply testing/libspatialite/APKBUILD]: I'm not intimately familiar with the project or the naming convention, but I do not think it's `alpha` as they have explicitly used `beta` before. Regardless, if `pkgver="4.3.0a"` works and is acceptable, I'll just do that. I was likely thinking that `pkgver` didn't accept characters, based on the issue I ran into with `testing/nitro` with `dev` being part of the version number.

**Pull request closed without merging**